i18n/en.json

```diff
@@ -4112,11 +3588,11 @@
   },
   {
     "id": "model.client.set_team_icon.no_file.app_error",
-    "translation": ""
+    "translation": "No file under 'image' in request"
```

Comment:
Which request?

Reply:
This is from the Go Client library. Probably here the error is not exactly that. Is more `"Unable to create the "image" field in the request"`. It tries to create the "image" field including the image to set as team icon, and if it fails for any reason, show this error.